GenderPayGap.WebUI/Views/ScopeNew/ConfirmInScope.cshtml

```diff
@@ -1,9 +1,107 @@
-﻿@{
-    ViewBag.Title = "Confirm In Scope - Gender pay gap service";
+﻿@using GovUkDesignSystem.GovUkDesignSystemComponents
+@using GovUkDesignSystem
+@using GenderPayGap.WebUI.Models.ScopeNew
+@using GenderPayGap.WebUI.Classes.Formatters
+@using GenderPayGap.Core
+@model GenderPayGap.WebUI.Models.ScopeNew.OutOfScopeViewModel
+
+@{
+    ViewBag.Title = "Confirm that your organisation is in scope - Gender pay gap service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";
+}
+
+@{
+    string encryptedOrganisationId = Encryption.EncryptQuerystring(Model.Organisation.OrganisationId.ToString());
+    string reportingYearsString = Model.ReportingYear.Year + "/" + (Model.ReportingYear.Year + 1).ToString().Substring(2, 2);
+}
+
+@section BeforeMain {
+    @{
+        var crumbs = new List<CrumbViewModel>
+        {
+            new CrumbViewModel
+            {
+                Text = "Your Organisations",
+                Href = Url.Action("ManageOrganisations", "Organisation")
+            },
+            new CrumbViewModel
+            {
+                Text = Model.Organisation.OrganisationName,
+                Href = Url.Action("ManageOrganisation", "Organisation", new {id = encryptedOrganisationId})
+            },
+            new CrumbViewModel
+            {
+                Text = "Change Scope (Reporting Year " + reportingYearsString + ")"
+            }
+        };
+    }
+
+    @(Html.GovUkBreadcrumbs(new BreadcrumbsViewModel
+    {
+        Crumbs = crumbs
+    }))
 }
+
+@{
+    string reportingYearFormatted = Model.ReportingYear.Year + "/" + (Model.ReportingYear.Year + 1).ToString().Substring(2, 2);
+    string snapshotDate = new GDSDateFormatter(Model.ReportingYear).FullStartDate;
+}
+
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-        Confirm in scope
+        <form method="POST" action="@Url.Action("ConfirmInScopeAnswers", "ScopeNew", new {encryptedOrganisationId, reportingYear = Model.ReportingYear.Year })">
+            @(Html.AntiForgeryToken())

```

Comment:
This is very minor:
I try to make the `<form>` surround the smallest possible set of elements - to differentiate between:
* data that will be send as part of the form
* data that is just being presented on the page for the user's information

So, this opening `<form>` tag could move down several lines, to just before the `@Html.GovUkButton` (make sure to keep the `@(Html.AntiForgeryToken())` within the form)